rustual-boy-cli/src/argparse.rs

```diff
@@ -0,0 +1,32 @@
+use clap::{App, Arg};
+
+pub struct CommandLineConfig {
+    pub rom_path: String,
+    pub sram_path: String,
+}
+
+pub fn parse_args() -> CommandLineConfig {
+    let app = App::new("Rustual-boy")
```

Comment:
I think we should use "Rustual Boy" without dashes whenever referring to the project; the dashes are more of a technical detail I think.